components/details/SharebuttonLinks.tsx

```diff
@@ -0,0 +1,177 @@
+// Copyright 2022 Curtin University
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author: Alex Massen-Hane
+
+import { useToast, Button, Portal, Text, Center, VStack, Flex, FlexProps, Box } from "@chakra-ui/react";
+import { Grid, GridItem } from "@chakra-ui/react";
+import MetadataLinkShare from "./MetadataLinkShare";
+import React, { memo } from "react";
+import { useClipboard } from "@chakra-ui/react";
+import { Popover, PopoverTrigger, PopoverContent, PopoverBody, PopoverArrow } from "@chakra-ui/react";
+import { FiShare2, FiLink2 } from "react-icons/fi";
+import { Entity } from "../../lib/model";
+
+interface SharebuttonLinksProps extends FlexProps {
+  entity: Entity;
+  category: string;
+  platform: string;
+  id: string;
+  hrefCoki: string;
+  iconTw: string;
+  iconFb: string;
+  iconLi: string;
+}
+
+// Needs to have the info on if it's region or institution for the tweet.
+const SharebuttonLinks = ({
+  entity,
+  category,
+  platform,
+  id,
+  hrefCoki,
+  iconTw,
+  iconFb,
+  iconLi,
+  ...rest
+}: SharebuttonLinksProps) => {
+  var shareText = encodeURIComponent("Open Access Research Performance for " + `${entity.name}` + "\n");
+
+  // TODO Somehow remove the domain name being hardcoded
+
+  if (hrefCoki === undefined) {
+    hrefCoki = "https://open.coki.ac";
+  } else {
+    hrefCoki = "https://open.coki.ac" + hrefCoki.slice(0, -1);
+  }
+
+  // Share page urls for each social site
+  const hrefTw = "https://twitter.com/intent/tweet?text=" + shareText + "&url=" + hrefCoki;
+  const hrefFb = "https://www.facebook.com/sharer/sharer.php?u=" + hrefCoki;
+  const hrefLi = "https://www.linkedin.com/shareArticle?mini=true&url=" + hrefCoki;
+
+  // Copy link button hook
+  const [value] = React.useState(hrefCoki);
+  const { hasCopied, onCopy } = useClipboard(value);
+
+  return (
+    <Popover>
+      <PopoverTrigger>
+        <Button variant="shareButton" data-test="Popover Share button">
+          <Flex paddingRight="10px" align="center">
+            <FiShare2 size={24} />
+          </Flex>
+          <Text casing="uppercase">Share</Text>
+        </Button>
+      </PopoverTrigger>
+      <Portal>
+        <PopoverContent
+          data-test={`${platform} share popover panel`}
+          width="min-content"
+          height="min-content"
+          _focus={{ border: "none" }}
```

Comment:
If you compare the popover to the default popover, the popover jiggles a bit and doesn't animate smoothly. I think it is because of the _focus={{ border: "none" }} tag, you might need to delve a little deeper into the theme to see how to turn off the blue outer border without disrupting the animation and creating the jiggling.